spinalcordtoolbox/scripts/sct_fmri_moco.py

```diff
@@ -63,6 +63,11 @@ def get_parser():
         help="Binary mask to limit voxels considered by the registration metric. You may also provide a softmask "
              "(nonbinary, [0, 1]), and it will be binarized at 0.5."
     )
+    optional.add_argument(
+        '-ref',
+        metavar=Metavar.file,
+        help="Reference volume for motion correction, for example the mean fMRI volume."
+    )
```

Comment:
It should be fairly easy to add a test for this new flag (as well as `num_target`), as well. :)